src/test/resources/application.properties

```diff
@@ -1,4 +1,15 @@
 spring.datasource.driver-class-name=org.h2.Driver
 spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
 spring.datasource.username=sa
-spring.datasource.password=sa
\ No newline at end of file
+spring.datasource.password=sa
+
+#spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False
```

Comment:
dlaczego to się pojawiło ?

Reply:
robiłem dodatkowe testy w MySql aby podejrzeć w workbenchu czy wszystkie id się zgadzają. Usunąć ?

Reply:
Usunięte